scrapers_next/ma/committees.py

```diff
@@ -0,0 +1,89 @@
+from spatula import URL, HtmlListPage, HtmlPage, CSS, XPath, selectors
```

Comment:
In addition to changing import of `selectors` to `SelectError` to import it directly, you will also need to import `SkipItem` from spatula for the changes I recommend below.

The reason we want to raise `SkipItem` for any committees without membership is because there needs to be at least one member in each scraped committee JSON file for the merge to our [people data repository](https://github.com/openstates/people/tree/main/data). We will run the scrapers again later in the session to get committee membership that was added later.